### ISSUE TITLE
[FP] The ka.ac.delay check is broken

### ISSUE THREAD

**Issue by SkyLicks** [jiangdashao/Matrix-Issues#879]
## Troubleshooting Information
`Change - [ ] to - [X] to check the checkboxes below.`
- [X] Matrix and ProtocolLib are up-to-date
- [X] Matrix is running on a 1.8, 1.12, 1.13, or 1.14 server
- [X] The issue happens on default config.yml and checks.yml
- [X] I've tested if the issue happens on default config

## Issue Information
**Server version**: 1.12.2
**Verbose messages**: https://youtu.be/IgmIuoUJLS4 and https://youtu.be/fN1Ga-TbSgw
**How/when does this happen**: To be honest, I am not sure exactly what causes it. However, in my tests this false positive only occured when I moved my mouse while hitting while BUTTERFLY clicking.
Link to checks.yml file: Default
Link to config.yml file: https://pastebin.com/cVeGSuyN
**Other information**: When I tested in the check would trigger with actual autoclicker, that wasn't detected as seen in https://github.com/jiangdashao/Matrix-Issues/issues/878. I believe ka.ac.delay is completely broken :/

## Configuration Files
Link to checks.yml file: Default
Link to config.yml file: https://pastebin.com/cVeGSuyN

**Comment by SkyLicks** [comment]
A player on my server reached like 30 or 40 violations by just clicking legit (or legit as far as I could tell).

**Comment by ZakShearman** [comment]
1) The video you show comes nowhere near to actually flagging.
2) Make sure you're on the latest version, because a lot of issues I found with the ka.ac.delay check were patched.

**Comment by SkyLicks** [comment]
1. True this video in particular doesnt. But this was just proof of concept. And as I mentioned above, my concerns relating to this component started after a player was false flagged severely.

2. I am on the latest version as indicated by the check marks under the "troubleshooting" section.

**Comment by ZakShearman** [comment]
Gonna try replicate, but chances are I won't be able to.

**Comment by SkyLicks** [comment]
Ok. Few things to keep in mind:

1. I was butterfly clicking.

2. I was in right left strafe motion.

**Comment by ZakShearman** [comment]
Could've been the butterfly clicking, I have no idea whether this anti cheat likes butterflying or not.

**Comment by ZakShearman** [comment]
Couldn't manage to replicate this one, but I did find an issue in another autoclicker check.

**Comment by SkillzGamingYT** [comment]
1. Nobody will ever click consistently that long.
2. The violations level aren't near level to be kicked (vl~10-15)
3. In #878 , The violation are way higher (vl25+), and display a message. 
4. Butterfly clicking should be considered as "risky" and "do it at your own risk".

I will agree that the devs should make the min CPS for consistent clicking check from 10 to like 15?

**Comment by SkyLicks** [comment]
> 1. Nobody will ever click consistently that long.
> 2. The violations level aren't near level to be kicked (vl~10-15)
> 3. In #878 , The violation are way higher (vl25+), and display a message.
> 4. Butterfly clicking should be considered as "risky" and "do it at your own risk".
> 
> I will agree that the devs should make the min CPS for consistent clicking check from 10 to like 15?

1. I don't think you have actually watched the video? In the video, we see I am NOT clicking consistently. My CPS is fluctuating according to the 5zig mod.
2. Well, in that video, not particularly... but if you read my above replies I mentioned another player got up to 40 VL. Here, I went a try x2 with this bug. This time I did trigger an alert: https://youtu.be/fN1Ga-TbSgw
3. Uhhh, I am reporting the ka.ac.delay check not working correctly. It has nothing to do with the ka.ac.cps-long-term check that happened at the end of the video xD. (Im sorry if I was unclear, I edited #878 to be more clear).
4. That would be up to @jiangdashao if he wants it to be like that, but most PvP servers allow butterfly clicking and I'd appreciate if we could keep up support for that <3.
